mrp_operations_rejected_quantity/wizard/mrp_work_order_produce.py

```diff
@@ -19,10 +32,30 @@ def default_get(self, var_fields):
             x.accepted_amount for x in
             operation.operation_time_lines.filtered(
                 lambda r: r.state == 'processed'))
-        res['product_qty'] = (operation.production_id.product_qty -
-                              accepted_amount)
-        res['qty_to_produce'] = operation.production_id.product_qty
-        res['accepted_amount'] = accepted_amount
+        pending_accepted_amount = sum(
```

Comment:
no se si es mejor hacerlo asi como lo has hecho o poner
`res['pending_accepted_amount'] = sum(x.accepted_amount for x in operation.operation_time_lines.filtered(lambda r: r.state == 'pending'))`
y luego hacer 
`res['pending_total_amount'] = res.get('pending_accepted_amount') + res.get('pending_rejected_amount') `
Que opinais el resto? @oihane @esthermm @Daniel-CA @mikelarre